utils_vad.py

```diff
@@ -122,16 +122,24 @@ def __call__(self, inputs: torch.Tensor):
 def read_audio(path: str,
                sampling_rate: int = 16000):
 
-    wav, sr = torchaudio.load(path)
+    if 'sox' in torchaudio.list_available_backends():
+        effects = [
+            ['channels', '1'],
```

Comment:
Hi, may i ask why do you choose channel-1, does it affect model performance if i choose to use channel-0 ?

Reply:
I believe this code is changing the audio from multiple channels (example: stereo) to a single channel, similarly to what the original code did, only more efficiently.